content/en/training/_index.html

```diff
@@ -0,0 +1,106 @@
+---
+title: Training
+bigheader: Kubernetes Training and Certification
+abstract: Training programs, certifications, and partners.
+class: gridPage
+cid: training
+---
+
+<div style="padding-top: 100px; padding-bottom: 100px; background-color: #3371e3; color:#fff"  >
+  <div class="main-section" style="width: 66%; margin-left: 16.5%;">
+    <center>
+      <h1>Build your cloud native career</h1>
+      <p style="color:#fff">Kubernetes is at the core of the cloud native movement. Training and certifications from the Linux Foundation and our training partners lets you invest in your career, learn Kubernetes, and make your cloud native projects successful.</p>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section" style="padding-top: 100px; padding-bottom: 100px">
+    <center>
+      <h2>Take a free course on edX</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Introduction to Kubernetes <br> &nbsp;</b>
+        </h5>
+        <p>Want to learn Kubernetes? Get an in-depth primer on this powerful system for managing containerized applications.</p>
+        <br><br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Introduction to Cloud Infrastructure Technologies</b>
+        </h5>
+        <p>Learn the fundamentals of building and managing cloud technologies directly from The Linux Foundation, the leader in open source.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Fundamentals of Containers, Kubernetes, and Red Hat OpenShift</b>
+        </h5>
+        <p>Learn practical techniques for creating and deploying containers on a Kubernetes cluster using Red Hat® OpenShift Container Platform.</p>
+        <br>
+        <a href="https://www.edx.org/course/fundamentals-of-containers-kubernetes-and-red-hat" target="_blank" class="button">Go to Course</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div style="padding-top: 100px; padding-bottom: 100px; background-color: #4c4c4c; color: #fff">
+  <div class="main-section" style="width: 66%; margin-left: 16.5%">
+    <center>
+      <h2>Learn with the Linux Foundation</h2>
```

Comment:
This section wasn't explicitly asked for, but it seems like the logical next step. However, it could become a bit of a sore point with other organizations that offer training. Either way I think we need to point people to something more fully-featured than the edX courses that could actually help them learn enough to get certified.

Thoughts?

Reply:
I think it's a fair addition, especially for a first iteration.

The shading in the div is confusing to me; at first I didn't realize there was more content below the EdX boxes, because the div shading looks like the footer at the bottom of the page. I like your idea (below) of a [Partners-style table](https://deploy-preview-19214--kubernetes-io-master-staging.netlify.com/partners/#kcsp), maybe organized by provider but not broken out by `<div>`--how feasible is that for a first implementation?

Reply:
As in: just the logos/links to providers? That would be relatively easy.

Reply:
Edit: The easiest/quickest way to do this ended up being adding the same table from the Partners page. It ends up being the most maintainable too, as the info is all pulled from the [cncf landscape](https://github.com/cncf/landscape) repo.